test/test_jit_cuda_fuser.py

```diff
@@ -885,6 +885,21 @@ def test_ternary_ops_type_promotion(self):
             self._ternary_test_helper(op, dtypes, True)  # random data
             self._ternary_test_helper(op, dtypes, False)  # special numbers
 
+    # We can't test the scalar version of rsub from python
```

Comment:
Out of curiosity: why can't we test scalar version of rsub from python?

Reply:
It says this in `native_functions.yaml`:

```
# For C++ only, until we have conversion from C++ numbers to Tensor
- func: sub.Scalar(Tensor self, Scalar other, Scalar alpha=1) -> Tensor
  device_check: NoCheck   # TensorIterator
  variants: function, method
  dispatch:
    CompositeExplicitAutograd: sub
```

I tried testing a scalar and it failed, just in case the comment was wrong.